### ISSUE TITLE
Allow to manually disable version manager detection

### ISSUE THREAD

**Issue by DaniruKun** [lexical-lsp/lexical#755]
I personally as an example use `nix` for everything, but for some projects I still use a version manager like `asdf`.
A lot of other language extensions allow you to specify a custom path to Elixir, or allow to completely disable the vm detection so it can pick up elixir from e.g. a Nix shell environment.

**Comment by scohen** [comment]
@Blond11516, I think that this will require changes on the lexical side as well, since version detection is done in the boot scripts.

**Comment by Blond11516** [comment]
Yeah that's why I transferred the issue here. There isn't much we can do on the client side to support this (not without changes to the boot scripts at least).

**Comment by scohen** [comment]
Out of curiosity, @DaniruKun, does this fail if you're using our flake? (I don't know much about nix)
